recipes/pytest-custom-exit-code/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pytest-custom-exit-code" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/92/9d/e1eb0af5e96a5c34f59b9aa69dfb680764420fe60f2ec28cfbc5339f99f8/pytest-custom_exit_code-{{ version }}.tar.gz"
```

Comment:
Please use the url as in the example recipe

Reply:
That one?
``https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz``
it was not working for this recipe

Reply:
https://pypi.io/packages/source/p/pytest-custom-exit-code/pytest-custom-exit-code-0.3.0.tar.gz

Reply:
https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/26740569#L271

Do you have any hints in this case?

Reply:
I think I know why, I will try to fix it, thanks

Reply:
https://pypi.io/packages/source/p/pytest-custom-exit-code/pytest-custom_exit_code-0.3.0.tar.gz

Reply:
Thanks!